server/src/test/java/org/elasticsearch/index/shard/IndexShardTests.java

```diff
@@ -2905,6 +2905,7 @@ public void testRecoverFromTranslog() throws IOException {
         closeShards(primary);
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/93168")
```

Comment:
This is fixed now, you can remove this I think.

Reply:
Sorry about that!